CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - Error messages in JSON output used to contain ANSI color codes,
   this was fixed so they are now uncolored (and readable!) plain text.
 - Fixed config parsing bug to cut CI scan speeds down significantly
+- Fixed parsing of CI environment variables for Circle CI, Bitbucket, and Buildkite
```

Comment:
This is the changelog of last week's release — could you add this entry as a file on changelog.d/ instead?

<img width="722" alt="CleanShot 2022-07-12 at 13 25 40@2x" src="https://user-images.githubusercontent.com/1060436/178588768-3fa9f7f4-77cb-45bc-81a1-e0bd4c75d7b5.png">